src/Tracing/SamplingContext.php

```diff
@@ -48,4 +54,20 @@ public function setParentSampled(?bool $parentSampled): void
     {
         $this->parentSampled = $parentSampled;
     }
+
+    /**
+     * @param array<string, mixed>|null $additionalContext
+     */
+    public function setAdditionalContext(?array $additionalContext): void
+    {
+        $this->additionalContext = $additionalContext;
+    }
+
+    /**
+     * @return array<string, mixed>|null
```

Comment:
Can you please add a little description of what this function does, e.g. `Gets additional optional context data.`?